src/headline-results/initial-measured-performance.md

```diff
@@ -30,17 +30,25 @@ relating to capability-enabled branch prediction, we were able to achieve a
 
 ## Result on improved Morello microarchitecture on FPGA
 
-Using a modified FPGA implementation of Morello developed after chip tapeout,
-to work around a costly microarchitectural data-dependent exception check,
-which had been highlighted by performance counter results from dynamic
-workloads, we were able to achieve a **7.40% geometric mean overhead for the
+Using modified FPGA implementations of Morello developed after chip tapeout,
+to work around a costly microarchitectural data-dependent exception check and
+store queue pressure, which had been highlighted by performance counter results
+from dynamic workloads, we were able to achieve **7.40% (solely working around
+the data-dependent check) and 5.70% (working around both the data-dependent
+check and increasing the store queue size) geometric mean overheads for the
 training workload of SPECint 2006 with "pure capability" code generation, with
-the Benchmark ABI, on an incrementally modified Morello design**.
+the Benchmark ABI, on incrementally modified Morello designs**.
 Analysis of the exception delivery issue suggests that more recent
 (post-Neoverse N1) microarchitectures already contain the changed behavior
-required to enable this optimization such that this roughly 51% of the
-overhead seen on the original Morello design (i.e., without the
+required to enable the data-dependence optimization such that this roughly 51%
+of the overhead seen on the original Morello design (i.e., without the
 data-dependency fix) could be decreased or eliminated.
+Store queue sizes in the original Morello design were not re-tuned in the
+presence of wider capabilities and were left tuned for baseline aarch64
+workloads, but a future optimized design would take such increased pressure
+into consideration.
+Re-tuning the Morello store queues achieves a further roughly 23% reduction in
```

Comment:
Do we think the store queue sizes used in the benchmarks are fully tuned or simply a significant improvement? If not do we want to hint at that here?

Reply:
Expanded rather than tuned, per se, and maybe a footnote clarifying that is useful?

Reply:
Isn't the usefulness rather implied by it reducing the overhead by 23%?